lib/views/staging-view.js

```diff
@@ -391,20 +400,61 @@ export default class StagingView {
         await this.showMergeConflictFileForPath(selectedItem.filePath, {activate: true});
       }
     } else {
-      const pendingItem = this.getPendingFilePatchItem();
-      if (openNew || pendingItem) {
-        const activate = pendingItem === this.props.workspace.getActivePaneItem();
-        await this.showFilePatchItem(selectedItem.filePath, this.selection.getActiveListKey(), {activate});
+      if (openNew) {
+        // User explicitly asked to view diff, such as via click
```

Comment:
👍 

I like explicitly separating the logic between "the user explicitly triggered this" and "we're inferring that this should happen."